packages/cli/src/commands/proxy/connect.ts

```diff
@@ -49,7 +53,7 @@ export default class Connect extends ProfileCommand<typeof Connect> {
     }
     const composeProject = args['compose-project']
     const deviceId = (await machineId(this.config.dataDir)).substring(0, 2)
-    const envId = normalizeEnvId(`${composeProject}-dev-${deviceId}`) // local+find_ambient_id_based on compose dir (?)
+    const envId = flags['env-id'] ? validateEnvId(flags['env-id']) : normalizeEnvId(`${composeProject}-dev-${deviceId}`)
```

Comment:
I would print a log describing how the id got created (if not specified explicitly), like the other commands do.

Reply:
I think it's a bit tricky to communicate concisely.
The current implementation is a mix of compose-project, dev suffix and deviceId.

I can use `generated` vs `user specified`, but I'm not sure if it reveals useful information as the user know they specify the id.

Reply:
Added log based on Docker Machine and `Local device id`